README.md

```diff
@@ -118,9 +118,20 @@ export default App;
 ```javascript 
 {
     [triggerChar: string]: {|
-        ?output: (item: Object | string, trigger?: string) => {| text: string, caretPosition: "start" | "end" | "next" | number |} | string,
-        dataProvider: (token: string) => Promise<Array<Object | string>> | Array<Object | string>,
-        component: ReactClass<*>,
+      output?: (
+        item: Object | string,
+        trigger?: string
+      ) =>
+        | {|
+            key?: string | number,
```

Comment:
I did this change but could you please change this as well to `?string` (it's the correct way how React defines `key` so we should stick with it as well as we discussed in different thread)